mission_control/navigator_launch/launch/master_boat.launch

```diff
@@ -2,5 +2,6 @@
     <include file="$(find navigator_launch)/launch/subsystems/hardware_drivers.launch"/>
     <include file="$(find navigator_launch)/launch/gnc.launch"/>
     <include file="$(find navigator_launch)/launch/subsystems/sensors.launch"/>
+    <include file="$(find navigator_launch)/launch/perception.launch" />
     <include file="$(find navigator_alarm)/launch/alarm.launch"/>
```

Comment:
@mattlangford is there a reason for alarm.launch not being in the navigator_launch pkg?